### PR TITLE
added new opensmile config files

### PR DESCRIPTION
**Pull request by jhagerer**
updated record rate to once an hour
updated README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,35 @@
 Audio plugin for the RADAR passive remote monitoring app. It uses openSMILE to process audio from
 the phone's microphone, extracts features from it, and sends those features. 
 
+# New Default RadarConfiguration Parameters #
+
+- "audio_duration": 15
```
Comment (MaximMoinat) [inline README.md]: Thanks for updating the readme. 
Could you create a table for the parameters with the unit, default and a description? For an example, see here: https://github.com/RADAR-base/radar-android-phone#sensors

**Review by MaximMoinat — APPROVED** [review]
Looks good to me. 
Next time, could you create a branch from this repo? You have got that access right now.

File: src/main/java/org/radarcns/audio/AudioServiceProvider.java [hunk 2]
```diff
@@ -37,9 +37,9 @@
     protected void configure(Bundle bundle) {
         super.configure(bundle);
         RadarConfiguration config = getConfig();
-        bundle.putLong(AUDIO_DURATION_S, config.getLong(AUDIO_DURATION_S, 5L));  // default: 5 second
-        bundle.putLong(AUDIO_RECORD_RATE_S, config.getLong(AUDIO_RECORD_RATE_S, 180L));  // default: 180 second
-        bundle.putString(AUDIO_CONFIG_FILE, config.getString(AUDIO_CONFIG_FILE, "liveinput_android.conf"));  // default: "liveinput_android.conf"
+        bundle.putLong(AUDIO_DURATION_S, config.getLong(AUDIO_DURATION_S, 15L));  // default: 5 second
```
Comment (MaximMoinat) [inline src/main/java/org/radarcns/audio/AudioServiceProvider.java]: The comments still contain the old defaults. Maybe just remove those

**Comment by jhagerer** [comment]
Please consider my changes regarding the inquired parameter table and removed comments. I hope you can forgive me, since I promise to use a separate branch next time.

**Pull request merged**